Cubical/Categories/RezkCompletion/Base.agda

```diff
@@ -0,0 +1,47 @@
+{-
+
+The Rezk Completion
+
+-}
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.RezkCompletion.Base where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Functor.ComposeProperty
+open import Cubical.Categories.Equivalence
+open import Cubical.Categories.Equivalence.WeakEquivalence
+open import Cubical.Categories.Instances.Functors
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' : Level
+    C : Category ℓC ℓC'
+    D : Category ℓD ℓD'
+
+-- Rezk Completion of a given category is the initial functor from it towards univalent categories.
+
+-- It's a bit technical to formulate the universal property of Rezk completion,
+-- because the universal property is naturally universal polymorphic,
+-- and so the predicate is not inside any universe of finite level.
+
+-- The product type with one parameter in Typeω
```

Comment:
Hmm, this is maybe the correct place for this? Not sure if it could be useful anywhere else though...

Reply:
Me neither. Definitely not the best place here, but I don't know a better place. Maybe `Cubical.Data.Prod`?

Reply:
Yeah, that would work. With a name involving `ω` or `Typeω`